VSTSDropProxy.cs

```diff
@@ -105,6 +108,7 @@ await Policy
                     })
                 .ExecuteAsync(async () =>
                 {
+                    //if we would set the timeout based on the size of the blob which is in the manifest.
```

Comment:
I do not understand the purpose or meaning of the new comment?

Reply:
slightly better.